src/marketing/variables-custom.md

```diff
@@ -4,38 +4,94 @@ redirect_from:
   - /system/variables-custom.html
 ---
 
-If you know a little basic HTML, you can create custom variables and use a [markup tag]({% link marketing/markup-tags.md -%}) to incorporate them into [email]({% link marketing/email-templates.md -%}) and [newsletter]({% link marketing/newsletter-template.md -%}) templates, as well as other types of [content]({% link cms/content-elements.md -%}).
+To meet the specific needs of your business, you can create custom variables and insert them into [pages]({% link cms/pages.md -%}), [blocks]({% link cms/blocks.md -%}) and [email]({% link marketing/email-template-message.md %}) templates. The list of whitelisted variables that appears when you click the Insert Variable button includes both [predefined]({% link marketing/variables-predefined.md %}) and custom variables. As shown in the following image, the list of available variables for a specific email template is determined by the data that is associated with the template. See the [Variable Reference]({% link marketing/variables-reference.md -%}) for a list of frequently-used email templates and their associated variables.
 
-![Custome variables]({% link images/images/variables-custom.png %}){: .zoom}
-_Custom Variables_
+{:.bs-callout-info}
+Only whitelisted [predefined]({% link marketing/variables-predefined.md %}) or custom variables can be used in email and newsletter templates.
```

Comment:
```suggestion
Only whitelisted [predefined]({% link marketing/variables-predefined.md %}) or [custom]({% link marketing/variables-custom.md %}) variables can be used in email and newsletter templates.
```

Reply:
This is OK as is -- it's not necessary to link to itself.